egs2/mini_an4/asr2/run.sh

```diff
@@ -4,11 +4,10 @@
 set -e
 set -u
 set -o pipefail
-
+# --kmeans_feature "mfcc" \
 ./asr2.sh \
     --nj 2 \
     --inference_nj 2 \
-    --kmeans_feature "mfcc" \
```

Comment:
recommend to use run.sh to change features instead of directly modify asr2.sh

Reply:
okay